pkg/networkmanager/firewall.jsx

```diff
@@ -143,7 +140,12 @@ class AddServicesBody extends React.Component {
 
     componentDidMount() {
         firewall.getAvailableServices()
-                .then(services => this.setState({ services: services }));
+                .then(services => this.setState({ services: services.map(s => {
+                    if (!s.name)
+                        s.name = s.id;
+                    return s;
+                })
+                }));
```

Comment:
A more succinct version of that is `map(s => s.name || s.id);`

Reply:
`s` is object that has other properties, this would only return one string, not edited object.
Best I could come with is:

    map(s => {                               
        s.name = s.name || s.id;                                
        return s;         
     })

Reply:
Oops, right. Nevermind then.